frontend/src/components/SearchArtist.js

```diff
@@ -0,0 +1,43 @@
+import styled from 'styled-components/macro'
+import { useState } from 'react'
+import { Link } from 'react-router-dom'
+
+export default function SearchArtist() {
+  const [artistSearchTerm, setArtistSearchTerm] = useState('')
+
+  return (
+    <SearchArtistContainer>
+      <form>
```

Comment:
currently your form should cause a page reload when someone hits enter, maybe you want to suppress the default behavior for your form somewhere

Reply:
pressing enter or using the search button is both fine for me. maybe I dont get ur point here ...?!

Reply:
You haven't specified an `onSubmit` tag in which you can define what happens when the user clicks the button or hits enter
The default behavior in HTML is a page reload which is usually avoided in react because it deletes useState values etc.
Therefore, you might want to consider suppressing the default behavior e.g. with `event.preventDefault()` and define your own function